models/app.js

```diff
@@ -49,27 +48,20 @@ function createModel (cb) {
     const dir = files[0]
     manager.create(dir, done)
   })
+
   model.effect('delete', (state, data, send, done) => {
     const dat = data
     manager.remove(dat, done)
   })
+
+  // copy a dat share link to clipboard and open a modal
```

Comment:
@juliangruber the effect has multiple behaviors; `share-` doesn't cover them all. `share-and-copy-link-to-clipboard` might be closer but that's too long. I feel a comment has added value here to describe the side effect that happens